src/Filter.ts

```diff
@@ -3,24 +3,24 @@ import * as ko from "knockout";
 // for country filters
 import {countries, pineCountries, gavi69, gavi73} from "./Data";
 
-export interface FilterOptions {
+export interface FilterSettings {
     name: string;
 }
 
-export interface RangeFilterOptions extends FilterOptions {
+export interface RangeFilterSettings extends FilterSettings {
     min: number;
     max: number;
     selectedLow?: number;
     selectedHigh?: number;
 }
 
-export interface ListFilterOptions extends FilterOptions {
+export interface ListFilterSettings extends FilterSettings {
     options: Array<string>;
```

Comment:
Renamed all these interfaces so we don't end up with the confusing `options.options`